src/components/input/color/index.tsx

```diff
@@ -1,5 +1,20 @@
-import { Component, Element, Event, EventEmitter, h, Host, Method, Prop, VNode } from "@stencil/core"
+import {
+	Component,
+	ComponentWillLoad,
+	Element,
+	Event,
+	EventEmitter,
+	h,
+	Host,
+	Listen,
+	Method,
+	Prop,
+	VNode,
+	Watch,
+} from "@stencil/core"
+import { SmoothlyInputCustomEvent } from "../../../components"
 import { Color } from "../../../model"
+import { SmoothlyInput } from ".."
```

Comment:
Maybe personal preference of me but i would prefer `../index`. In this case it does not matter but `../index` will always work. `..` might misbehave if there is a `package.json` in the target directory. You will for the most part notice this issue when writing tests for other utily libraries.

Reply:
it was auto imported so i didnt even see that! I agree